client/src/main/java/tds/assessment/Assessment.java

```diff
@@ -215,4 +216,19 @@ public boolean isInitialAbilityBySubject() {
     public void setInitialAbilityBySubject(boolean initialAbilityBySubject) {
         this.initialAbilityBySubject = initialAbilityBySubject;
     }
+
+    /**
+     * @return A collection of languages that are associated with this {@link tds.assessment.Assessment}
+     * <p>
+     *     The languages that are available to an assessment are determined by collecting the unique "language"
+     *     {@link tds.assessment.ItemProperty} of the {@link tds.assessment.Item}.
+     * </p>
+     */
+    public Set<String> getLanguages() {
+        return languages;
+    }
+
+    public void setLanguages(Set<String> languages) {
```

Comment:
Is this the "language code" or the "language"?  I remember @burzum619 recommending a naming convention that was language code when we were using ENU or ESN.  If we are following that practice, does this need to chagne to `getLanguageCodes` and `setLanguageCodes`?

Update: As I got further down, I see that `languageCode` is used as the parameter to `getForm()`, so I'm assuming this is the languageCode and should be renamed.